### PR TITLE
Explore – Web

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

**Comment by alisa911** [comment]
https://github.com/osmandapp/OsmAnd-Issues/issues/2516

File: map/src/menu/search/PhotoGallery.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+export default function PhotoGallery({ photos }) {
+    const [open, setOpen] = useState(false);
+    const [activeStep, setActiveStep] = useState(0);
+    const { t } = useTranslation();
+
+    const [loading, setLoading] = useState(true);
+
+    const handleImageLoad = () => {
+        setLoading(false);
+    };
```
Comment (RZR-UA) [inline map/src/menu/search/PhotoGallery.jsx:17]: Could be 1-liner

File: map/src/menu/search/PhotoGallery.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        const imageExtensions = ['.jpeg', '.jpg', '.png', '.gif'];
+        return photos.features.filter((photo) => {
+            const extension = photo.properties.imageTitle
+                .slice(photo.properties.imageTitle.lastIndexOf('.'))
+                .toLowerCase();
+            return imageExtensions.includes(extension);
+        });
+    }
+
+    const filteredPhotos = filterPhotos(photos).slice(0, 100);
```
Comment (RZR-UA) [inline map/src/menu/search/PhotoGallery.jsx:33]: Max 100 photos? Please create const.

**Review by RZR-UA — CHANGES_REQUESTED** [review]
Finally, please replace `http://` with `https://` in WptTagsProvider.js and WptDetails.jsx and verify the result.

**Review by RZR-UA — APPROVED** [review]

**Pull request merged**